### ISSUE TITLE
test wfs-1.1.0-Transaction-tc13.1 uses invalid typeName

### ISSUE THREAD

**Issue by segoerke** [opengeospatial/ets-wfs11#22]
Problem is empty Typename Element in request:

```
Test wfs:GetFeatureByName type Mandatory default result Passed (s0034/d150e34518_1/d150e721_1/d150e31216_1/d150e25935_1/d150e28647_1)

Assertion: Attempt to fetch a feature by name using the GetFeature/POST binding.

Request d150e31373_1:
   Method: POST
   URL: http://demo.luciad.com:8080/LuciadLightspeedOGC2014/wfs?
   Body:
<foo:GetFeature xmlns:foo="http://www.opengis.net/wfs" xmlns="http://www.w3.org/2001/XMLSchema"
                xmlns:ctl="http://www.occamlab.com/ctl"
                xmlns:gen="java:com.occamlab.te.Generator"
                xmlns:gml="http://www.opengis.net/gml"
xmlns:myparsers="http://teamengine.sourceforge.net/parsers"
                xmlns:ows="http://www.opengis.net/ows"
                xmlns:p="http://teamengine.sourceforge.net/parsers"
                xmlns:parsers="http://www.occamlab.com/te/parsers"
                xmlns:saxon="http://saxon.sf.net/"
                xmlns:sf="http://cite.opengeospatial.org/gmlsf"
                xmlns:te="http://www.occamlab.com/te"
                xmlns:tec="java:com.occamlab.te.TECore"
                xmlns:tems="java:com.occamlab.te.web.MonitorServlet"
                xmlns:wfs="http://www.opengis.net/wfs"
                xmlns:xi="http://www.w3.org/2001/XInclude"
                xmlns:xlink="http://www.w3.org/1999/xlink"
                xmlns:xs="http://www.w3.org/2001/XMLSchema"
                xmlns:xsd="http://www.w3.org/2001/XMLSchema"
                service="WFS"
                version="1.1.0">
   <foo:Query typeName="">
      <ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">
         <ogc:PropertyIsEqualTo>
<ogc:PropertyName>gml:name</ogc:PropertyName>
            <ogc:Literal/>
         </ogc:PropertyIsEqualTo>
      </ogc:Filter>
   </foo:Query>
</foo:GetFeature>
   Response from parser p:XMLValidatingParser.GMLSF1:
      <ExceptionReport xmlns="http://www.opengis.net/ows" xmlns:ows="http://www.opengis.net/ows"
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                 version="1.1.0"
                 xsi:schemaLocation="http://www.opengis.net/ows http://schemas.opengis.net/ows/1.0.0/owsExceptionReport.xsd">
   <ows:Exception exceptionCode="Invalid type name" locator="typeName">
      <ows:ExceptionText>

No feature type set.

</ows:ExceptionText>
   </ows:Exception>
   <!--Response received in [139] milliseconds--></ExceptionReport>

Message d150e31414_1:
   FAILURE: Expected valid wfs:FeatureCollection in response.

Result: Failed
```

**Comment by dbretschneider** [comment]
I have executed the actual ```ets-wfs11``` branch ```master```. I cannot reproduced the error. The test step was succesful with all sub-tests.

![tc13-1](https://cloud.githubusercontent.com/assets/3016561/7131900/70ee4824-e284-11e4-8f7b-8ab8df967626.png)

**Comment by dbretschneider** [comment]
@segoerke: Can we close this issue? I cannot reproduced.

**Comment by dbretschneider** [comment]
Cannot reproduced, because deegree used HTTP GET. Issue occurs while using HTTP POST.

**Comment by bermud** [comment]
Was closed for 1.23
